Source/WebKit/WebProcess/GPU/webrtc/LibWebRTCCodecs.cpp

```diff
@@ -106,6 +106,17 @@ void LibWebRTCCodecs::setHasVP9ExtensionSupport(bool hasVP9ExtensionSupport)
 
 #endif
 
+std::optional<VideoCodecType> LibWebRTCCodecs::videoEncoderTypeFromWebCodec(const String& codec)
+{
+    if (codec.startsWith("avc1."_s))
+        return VideoCodecType::H264;
+
+    if (codec.startsWith("hev1."_s) || codec.startsWith("hvc1."_s))
```

Comment:
Shouldn't these be case insensitive comparisons?

Reply:
This is the WebCodecs code path which is case sensitive.
I agree it is a bit confusing, I'll add a comment.